### DISCUSSION TITLE
What happened to the initial git remote setup code that was displayed on new repo creation?

### DISCUSSION THREAD

**Discussion by TwoFistedJustice** [community/community#60902] [Repositories] [answered]
### Select Topic Area

Question

### Body

Until very recently, a few weeks ago I think, when I created a new repo, Github provided some copy + paste code to easily connect my local repo to their hosted repo using either https or ssh. Something like:

`git remote add origin git@github.com/me/myrepo.git`

Tonight when I made a new repo I couldn't find any such code. I had to *gasp*! look it up! 

It was really helpful to be able to copy and paste it and be up and coding in 20 seconds and not have spend time looking it up.

Did I somehow just not see the code on the site? or has it been removed?

**Comment by the-officialjosh** [comment, marked as answer]
Well I just tried now and it is still there, It appears maybe you added other files to the new repo like a readme or gitignore

**Reply by TwoFistedJustice** [reply]
I did choose a license, which I normally do not. So that must be it. Thank you!

**Reply by the-officialjosh** [reply]
you can add the license after you connect the repo
